spec/timescaledb/rails/hyperfunctions_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+require 'spec_helper'
+
+describe Timescaledb::Rails::Hyperfunctions do
+  describe '.time_bucket' do
+    let(:one_day_old_event) do
+      Event.create(name: 'Test Event 1', occurred_at: 1.day.ago, recorded_at: 1.day.ago, created_at: 1.day.ago)
+    end
+
+    let(:ten_days_old_event) do
+      Event.create(name: 'Test Event 2', occurred_at: 10.days.ago, recorded_at: 10.days.ago, created_at: 10.days.ago)
+    end
+
+    after { Event.delete_all }
+
+    before do
+      one_day_old_event
+      ten_days_old_event
+    end
+
+    context 'when the date column is not specified' do
```

Comment:
Let's write a few specs to ensure that it raises an error if you pass a wrong interval or if you pass a wrong time column name